platform/iphone/Rtt_IPhoneDevice.mm

```diff
@@ -393,7 +393,9 @@
 		case MPlatformDevice::kLocationEvent:
 		case MPlatformDevice::kHeadingEvent:
 		case MPlatformDevice::kMultitouchEvent:
-		case MPlatformDevice::kKeyEvent:
+        case MPlatformDevice::kKeyEvent:
+            hasEventSource = true; //There is not reliable way to check this
```

Comment:
It seems this was added [around](https://developer.apple.com/documentation/uikit/uikeyboardhidusage/uikeyboardhidusagekeyboarda?language=objc) iOS13.3, so may be some check like
 `&UIKeyboardHIDUsageKeyboardA != NULL` would work. Also, got to test on the older iOS device

Reply:
Just added 
`if (@available(iOS 13.4, *)) { //Key Events only work on iOS 13.4 >=
                hasEventSource = true;
}`

I tried &UIKeyboardHIDUsageKeyboardA != NULL but getting error "Cannot take the address of an rvalue"